workspaces/contact/src/UnlitContact.class.ts

```diff
@@ -1,21 +1,27 @@
 // Import Third-party Dependencies
 import type { Contact } from "@nodesecure/npm-types";
+import type { RequireAtLeastOne } from "type-fest";
 
 // Import Internal Dependencies
 import * as utils from "./utils/index.js";
 
-export type IlluminatedContact = Contact & {
+export type ExtendedContact = RequireAtLeastOne<
```

Comment:
```suggestion
export type EnforcedContact = RequireAtLeastOne<
```
```suggestion
export type RestrictedContact = RequireAtLeastOne<
```

`Extended` can be a bit confusing / is not very accurate IMO
It suggest we are extending `Contact`, while we are enforcing the presence of `name` or `email` (this is how i saw it, at least 😄)

WDYT ?